meson_options.txt

```diff
@@ -19,6 +19,7 @@ option('plugin_uefi', type : 'boolean', value : true, description : 'enable UEFI
 option('plugin_nvme', type : 'boolean', value : true, description : 'enable NVMe support')
 option('plugin_modem_manager', type : 'boolean', value : false, description : 'enable ModemManager support')
 option('plugin_flashrom', type : 'boolean', value : false, description : 'enable libflashrom support')
+option('plugin_coreboot', type : 'boolean', value : false, description : 'enable coreboot support')
```

Comment:
Since this doesn't (currently) introduce any new dependencies, what would you think about just enabling it automatically when plugin_flashrom is set?